pkg/model/health.go

```diff
@@ -0,0 +1,94 @@
+package model
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/Merteg/pl-health-service/proto"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+type Health struct {
+	Id         string             `json:"_id,omitempty", bson: "_id, omitempty"`

```

Comment:
Tags should be space separated, not comma separated
ie 
`json:"_id,omitempty" bson:"_id,omitempty"` instead of `json:"_id,omitempty", bson: "_id, omitempty"`